source/octf/trace/parser/ParsedIoTraceEventHandler.h

```diff
@@ -70,11 +67,20 @@ class ParsedIoTraceEventHandler
 
     void flushEvents();
 
+    void pushOutEvent(proto::trace::ParsedEvent &event);
+
 private:
-    std::map<uint64_t, proto::trace::ParsedEvent> m_cache;
+    struct Key;
+    class Map;
+    struct IoQueueDepth;
+    std::queue<proto::trace::ParsedEvent> m_queue;
+    std::unique_ptr<Map> m_lbaMapping;
```

Comment:
I don't think this is a proper name - we map by lba, len and devid. I suggest eventMapping